src/hotspot/share/runtime/objectMonitor.inline.hpp

```diff
@@ -55,14 +55,18 @@ inline jint ObjectMonitor::waiters() const {
 
 // Returns NULL if DEFLATER_MARKER is observed.
 inline void* ObjectMonitor::owner() const {
-  void* owner = _owner;
+  void* owner = owner_raw();
```

Comment:
There is an inconsistency with `owner()` and `owner_is_DEFLATER_MARKER()` where one delegates to `owner_raw()` and the other repeats the `Atomic::load(&_owner)`. They should both do the same thing - probably better to use `owner_raw()`.

Reply:
Good catch! I changed owner_is_DEFLATER_MARKER() to use owner_raw().